dev-python/protobuf-python/protobuf-python-4.21.5.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2008-2022 Gentoo Authors
```

Comment:
GENTOO_MIRRORS="" ebuild protobuf-python-4.21.5.ebuild clean fetch
Appending /var/db/repos/vaukai to PORTDIR_OVERLAY...
`>>>` Downloading 'https://github.com/protocolbuffers/protobuf/archive/v21.5.tar.gz'

Maybe better rename it `4.21.5` -> `21.5` 
and have it as `dev-python/protobuf-python-21.5.ebuild` 
with `SRC_URI="https://github.com/protocolbuffers/protobuf/archive/v${PV}.tar.gz"`

Reply:
It's better to keep for Protobuf library and its binding the same archive, it avoids duplication of the archive, saving storage and time for download.
And why the full Python version instead of its parent's, because this is the declared version in project's file [version.json](https://github.com/protocolbuffers/protobuf/blob/v21.5/version.json), and also the expected version on [Pypi](https://pypi.org/project/protobuf/4.21.5/).